src/plugins/discover/public/application/components/doc_viewer_links/doc_viewer_links.tsx

```diff
@@ -32,6 +36,7 @@ export function DocViewerLinks(renderProps: DocViewLinkRenderProps) {
             target="_blank"
             style={{ fontWeight: 'normal' }}
             data-test-subj={`${item['data-test-subj']}-${index}`}
+            aria-label={item['aria-label']}
```

Comment:
This link has a concise text in it and shouldn't need `aria-label`; if you have to add, just use the `item.label`.

Reply:
Removed the aria-label